rustls-platform-verifier/Cargo.toml

```diff
@@ -52,11 +52,11 @@ webpki-roots = "0.26"
 [target.'cfg(target_os = "freebsd")'.dev-dependencies]
 webpki-roots = "0.26"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
+[target.'cfg(any(target_os = "macos", target_os = "ios", target_os = "tvos"))'.dependencies]
 core-foundation = "0.9"
 core-foundation-sys = "0.8"
-security-framework = { version = "2.6", features = ["OSX_10_14"] }
-security-framework-sys = { version = "2.4", features = ["OSX_10_14"] }
+security-framework = { version = "2.10", features = ["OSX_10_14"] }
+security-framework-sys = { version = "2.10", features = ["OSX_10_14"] }
```

Comment:
The upstream is a bit messy here :-/ No GitHub releases or changelog. Cargo.toml version updates are rolled in to other commits, etc.

I'm assuming this is for https://github.com/kornelski/rust-security-framework/pull/190?

Reply:
Yes the version bump is for those updates.

Otherwise I get the follwing import compile errors
`nresolved imports 'security_framework::certificate', 'security_framework::policy', 'security_framework::secure_transport', 'security_framework::trust'`

Reply:
For the curious:

* [diff.rs for -sys 2.4.0 -> 2.1.0](https://diff.rs/security-framework-sys/2.4.0/2.10.0/Cargo.toml)
* [diff.rs for security-framework 2.6.0 -> 2.10.0](https://diff.rs/security-framework/2.6.0/2.10.0/Cargo.toml)

Reply:
TIL about `diff.rs`, that's a very useful website.